cmd/deployctl/cmd/build.go

```diff
@@ -316,6 +316,8 @@ func CollectCmdRun(cmd *cobra.Command, args []string) {
 
 	platformNetworkFilters := make([]build.PlatformNetworkFilter, 0)
 
+	platformNetworkFilters = append(platformNetworkFilters, build.NewControllerAddressFilter())
```

Comment:
Default filter should not be append like that, check NewStorageMonitorFilter in defaultHostFilters